### ISSUE TITLE
Unclear sentence in Introduction "As the described solution ... requests only."

### ISSUE THREAD

**Issue by EskoDijk** [anima-wg/anima-brski-prm#25]
> As the described solution will rely on additional wrapping signature it will require pre-processing specifically for EST. EST simpleenroll uses PKCS#10 requests only.

This was unclear to me and can be puzzling to a reader that just starts reading the intro without yet knowing the rest of the document. And the last few words were overly detailed for an intro. Maybe rephrase:

> So instead of using TLS to provide secure transport between the pledge and the registrar-agent, the BRSKI-PRM solution will rely on an additional wrapping signature on the (EST) enrollment request by the pledge. The registrar then needs to do additional pre-processing to verify this signature, which is not present in classic EST.

**Comment by stfries** [comment]
Agree in general. I have change the proposed text a litte to: 
Instead of using TLS to provide secure transport between the pledge and the registrar-agent, BRSKI-PRM will rely on an additional wrapping signature of the enrollment request by the pledge. For EST {{RFC7030}} the registrar then needs to do additional pre-processing by verifying this signature, which is not present in EST.
